programs/past-programs.md

```diff
@@ -3,6 +3,15 @@ layout: markdown
 hero-image: url(../assets/images/stock-images/prep.jpg)
 hero-message: Past Programs
 ---
+# Fall 2016 Workshops
+
+## Building Bridges with Pioneers in Engineering
+
+The first in our series of Fall Workshops, Building Bridges with Pioneers in Engineering took place on campus on Saturday, September 24th from 9:45 AM to 3:00 PM in O'Brien Breezeway. Students from all backgrounds and experiences had tool training, learned about prototyping, design, and participated in designing and constructing their own bridge.
+
+## Creating Code with PiE
+PiE hosted its second workshop at UC Berkeley on Saturday, September 29th from 10 AM to 3:00 PM in Soda Hall. While our workshop is intended for students with no programming experience, students from all backgrounds and experiences are appreciated and very welcome! At this workshop, students learned about basic programming logic, variables, conditionals and loops, functions, and get a taste of recursion -- all in Scratch! All computers were provided.
```

Comment:
This was for... high school or UCB students? Or both came? If so, maybe write "students from all backgrounds and experiences were welcomed, including high school students and UCB students". Also, consistent past tense is preferred (imo).

Reply:
The workshops were only for high school students

Reply:
Will change all to past tense!